### ISSUE TITLE
Linking MIMIC-IV and MIMIC-CXR, under the {subject_id, hadm_id, stay_id} tuple key

### ISSUE THREAD

**Issue by ArbelHaiman** [MIT-LCP/mimic-code#1552]
Hello,

I want to create a dataset, comprising of records defined by the {subject_id, hadm_id, stay_id} identifier, i.e., each unique tuple defines one record. I looked at the MIMIC-CXR, but I saw that the columns hadm_id and stay_id do not exist there. I saw that there are only the subject_id and the study_id columns. How can I link only the relevant images to each record?

**Comment by alistairewj** [comment]
Yes, there is no explicit link between MIMIC-CXR and `hadm_id` / `stay_id`.

If you use a combination of the DICOM `StudyDate`, `StudyTime`, and `PatientID` (which links to `subject_id`), you can assign each x-ray to an `hadm_id` or `stay_id`.
